change/@fluentui-react-search-ef1e9132-0d99-40b3-8598-04d52166921f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32114&commitId=c66082190b1fc6aad060b90fc70d8296fc497a20&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|2| Changed|

</details>


  <details><summary><b>SearchBox Converged</b> 22 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SearchBox Converged.Appearance- filled-lighter.default.chromium.png|857| Changed|
|SearchBox Converged.Clears value.default.chromium.png|631| Changed|
|SearchBox Converged.Disabled, appearance- filled-darker.default.chromium.png|625| Changed|
|SearchBox Converged.Disabled, appearance- filled-darker.hover.chromium.png|625| Changed|
|SearchBox Converged.Disabled, appearance- outline.default.chromium.png|625| Changed|
|SearchBox Converged.Disabled, appearance- outline.focused.chromium.png|625| Changed|
|SearchBox Converged.Appearance- outline (default).default.chromium.png|631| Changed|
|SearchBox Converged.Disabled, appearance- underline.default.chromium.png|623| Changed|
|SearchBox Converged.Appearance- filled-darker.default.chromium.png|619| Changed|
|SearchBox Converged.Size- large.default.chromium.png|727| Changed|
|SearchBox Converged.With custom dismiss.default.chromium.png|631| Changed|
|SearchBox Converged.With value.default.chromium.png|631| Changed|
|SearchBox Converged.Disabled, appearance- underline.hover.chromium.png|623| Changed|
|SearchBox Converged.Appearance- underline.default.chromium.png|627| Changed|
|SearchBox Converged.Disabled, appearance- filled-lighter.focused.chromium.png|865| Changed|
|SearchBox Converged.Disabled, appearance- filled-darker.focused.chromium.png|625| Changed|
|SearchBox Converged.Size- small.default.chromium.png|536| Changed|
|SearchBox Converged.Disabled, appearance- underline.focused.chromium.png|623| Changed|
|SearchBox Converged.Disabled, appearance- filled-lighter.hover.chromium.png|865| Changed|
|SearchBox Converged.Disabled, appearance- filled-lighter.default.chromium.png|865| Changed|
|SearchBox Converged.Disabled, appearance- outline.hover.chromium.png|625| Changed|
|SearchBox Converged.With custom dismiss - RTL.default.chromium.png|844| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>